programs/ssl/ssl_client2.c

```diff
@@ -1534,13 +1534,18 @@ int main( int argc, char *argv[] )
             {
                 sig_alg_list[i++] = MBEDTLS_TLS13_SIG_ECDSA_SECP521R1_SHA512;
             }
+            else if( strcmp( q, "rsa_pkcs1_sha256" ) == 0 )
```

Comment:
Add Macro wrapper

Reply:
I Just change back to align with the exist coding style as comments https://github.com/ARMmbed/mbedtls/pull/5127#discussion_r755114957.  Please help check whether we can close this conversation.